### ISSUE TITLE
Docs for local and utc methods need rewording

### ISSUE THREAD

**Issue by mj1856** [moment/momentjs.com#227]
The current docs for the [`local`](http://momentjs.com/docs/#/manipulating/local/) and [`utc`](http://momentjs.com/docs/#/manipulating/utc/) functions are worded as:

> Sets a flag on the original moment to internally use ...

An end-user should not care at all what happens internally.  These are public APIs, and should be documented according to what public-facing functionality they provide.  Specifically, they adjust the output of the moment object to either the local time zone or to UTC, while retaining the same internal instant in time.

**Comment by mj1856** [comment]
Note there is similar wording in the [`moment.utc()`](http://momentjs.com/docs/#/parsing/utc/) docs.

**Comment by mikescott8** [comment]
submitted a pull request today on May 2nd for rewording the given text.

**Comment by mikescott8** [comment]
from initial pull request additional information to add to the documentation:

First of all, 'parse or display' really isn't right at this point in the docs. The date has already been parsed, so it's just a display thing.

Also, in addition to local and UTC mode, moment can also run in a fixed offset mode:

```
 moment.parseZone('2016-05-03T22:15:01+02:00').format()
 "2016-05-03T22:15:01+02:00"
``` 

Local and UTC can be used to convert out of that fixed offset mode:

```
moment.parseZone('2016-05-03T22:15:01+02:00').local().format()
"2016-05-03T15:15:01-05:00"

moment.parseZone('2016-05-03T22:15:01+02:00').utc().format()
"2016-05-03T20:15:01Z"

```

**Comment by maggiepint** [comment]
Merged in #299
